lib/features/history/models/income_item_uimodel.dart

```diff
@@ -6,17 +6,18 @@ import 'package:givt_app_kids/helpers/app_theme.dart';
 import 'package:givt_app_kids/helpers/datetime_extension.dart';
 
 class IncomeItemUIModel {
-  const IncomeItemUIModel({required this.income});
+  const IncomeItemUIModel({required this.income, required this.name});
 
   final Income income;
+  final String name;
 
   String get leadingSVGAsset => 'assets/images/donation_states_added.svg';
   double get amount => income.amount;
   Color get amountColor => const Color(0xFF06509B);
   bool get amountShowPlus => true;
   String get title => income.type == HistoryTypes.topUp
```

Comment:
both should have the same text

Reply:
"When money is added to the wallet (Recurring & top up) we show - 'Awesome! Your parents added more to your Wallet"

Reply:
oh